crates/web5/src/jose.rs

```diff
@@ -26,54 +26,19 @@ pub struct JoseHeader {
 impl FromJson for JoseHeader {}
 impl ToJson for JoseHeader {}
 
-#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
-pub struct JwtClaims {
-    #[serde(skip_serializing_if = "Option::is_none")]
-    pub iss: Option<String>,
-    #[serde(skip_serializing_if = "Option::is_none")]
-    pub jti: Option<String>,
-    #[serde(skip_serializing_if = "Option::is_none")]
-    pub sub: Option<String>,
-    #[serde(
-        serialize_with = "serialize_optional_unix_timestamp",
-        deserialize_with = "deserialize_optional_unix_timestamp",
-        skip_serializing_if = "Option::is_none",
-        default
-    )]
-    pub nbf: Option<SystemTime>,
-    #[serde(
-        serialize_with = "serialize_optional_unix_timestamp",
-        deserialize_with = "deserialize_optional_unix_timestamp",
-        skip_serializing_if = "Option::is_none",
-        default
-    )]
-    pub iat: Option<SystemTime>,
-    #[serde(
-        serialize_with = "serialize_optional_unix_timestamp",
-        deserialize_with = "deserialize_optional_unix_timestamp",
-        skip_serializing_if = "Option::is_none",
-        default
-    )]
-    pub exp: Option<SystemTime>,
-    #[serde(flatten, skip_serializing_if = "Option::is_none")]
-    pub additional_properties: Option<HashMap<String, JsonValue>>,
-}
-
-impl FromJson for JwtClaims {}
-impl ToJson for JwtClaims {}
-
-pub struct Jwt {
+pub struct Jws {
```

Comment:
oh btw @diehuxx @nitro-neal forgot to mention earlier, but I've also added `Jws` here b/c tbdex uses a detatched JWS for message signatures